src/lsst/rsp/__init__.py

```diff
@@ -6,7 +6,7 @@
 
 from .catalog import get_catalog, get_tap_service, retrieve_query
 from .forwarder import Forwarder
-from .logging import IPythonHandler, forward_lsst_log
+from .log import IPythonHandler, forward_lsst_log
```

Comment:
Why this renaming? Wouldn't this break any user of the library that was importing these functions under the old name?

Reply:
It was actually breaking as it was, because "logging" was getting shadowed and that was choking farther on.

I think the only real consumer of this is the 20-logging.python, and it imports from the top-level lsst.rsp rather than lsst.rsp.logging, which is how I would prefer people use it.

Reply:
It shouldn't be possible in Python 3 to have `lsst.rsp.logging` shadow the Python `logging` module because absolute imports are now the default. We use `safir.logging`, for example, without any trouble. But it sounds like the chances of this breaking something are fairly low.